tests/test_paperqa.py

```diff
@@ -3108,3 +3108,31 @@ async def test_reader_config_propagation(stub_data_dir: Path) -> None:
     assert mock_read_doc.call_args.kwargs["chunk_chars"] == 2000
     assert mock_read_doc.call_args.kwargs["overlap"] == 50
     assert mock_read_doc.call_args.kwargs["dpi"] == 144
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("filename", ["dummy.docx", "dummy.pptx", "dummy.xlsx"])
+async def test_parse_office_doc(stub_data_dir: Path, filename: str) -> None:
+    file_path = stub_data_dir / filename
+    if not file_path.exists():
+        pytest.skip(f"{filename} not found in stub_data")
+
+    docs = Docs()
+
+    settings = Settings(
+        llm="gemini/gemini-2.5-flash",
+        embedding="gemini/text-embedding-004",
+        summary_llm="gemini/gemini-2.5-flash",
+        agent={"agent_llm": "gemini/gemini-2.5-flash"},
+        parsing=ParsingSettings(use_doc_details=False, disable_doc_valid_check=True),
```

Comment:
```suggestion
        parsing=ParsingSettings(use_doc_details=False),
```

These docs should be valid (we don't need `disable_doc_valid_check=True`)